make get_data_editor_prebuilt_api_dir() available for !TOOLS_ENABLED
build

Index: modules/mono/godotsharp_dirs.h
--- modules/mono/godotsharp_dirs.h.orig
+++ modules/mono/godotsharp_dirs.h
@@ -47,6 +47,8 @@ String get_res_temp_assemblies_dir();
 String get_mono_user_dir();
 String get_mono_logs_dir();
 
+String get_data_editor_prebuilt_api_dir();
+
 #ifdef TOOLS_ENABLED
 String get_mono_solutions_dir();
 String get_build_logs_dir();
@@ -56,7 +58,6 @@ String get_project_sln_path();
 String get_project_csproj_path();
 
 String get_data_editor_tools_dir();
-String get_data_editor_prebuilt_api_dir();
 #else
 String get_data_game_assemblies_dir();
 #endif
